content/webapp/pages/exhibition-guide.tsx

```diff
@@ -66,31 +69,20 @@ const ExhibitionGuidesPage: FC<Props> = ({
       siteSection={'whats-on'}
       image={undefined} // TODO, linked doc promo image, e.g. Exhibition image
     >
-      <p>Wellcome Collection Exhibition guide: {exhibitionGuide.title}</p>
-      <pre
-        style={{
-          maxWidth: '600px',
-          margin: '0 auto 24px',
-          fontSize: '14px',
-        }}
-      >
-        <code
-          style={{
-            display: 'block',
-            padding: '24px',
-            backgroundColor: '#EFE1AA',
-            color: '#000',
-            border: '4px solid #000',
-            borderRadius: '6px',
-          }}
-        >
-          <details>
-            <summary>exhibition-guide</summary>
-            {/* eslint-disable-next-line no-restricted-syntax */}
-            {JSON.stringify(exhibitionGuide, null, 1)}
-          </details>
-        </code>
-      </pre>
+      <Layout10>
+        <Space v={{ size: 'xl', properties: ['margin-top'] }}>
+          <h2>{exhibitionGuide.title || ''}</h2>
```

Comment:
Can there ever not be a title? If so, I'd make the whole `h2` conditional (but I'd have thought it was a requirement, probably)